integrations/java/iceberg-1.2/openhouse-java-runtime/src/main/java/com/linkedin/openhouse/javaclient/OpenHouseTableOperations.java

```diff
@@ -299,6 +299,8 @@ static Mono<GetTableResponseBody> handleCreateUpdateHttpError(
       return Mono.error(
           new BadRequestException(
               casted, casted.getStatusCode().value() + " , " + casted.getResponseBodyAsString()));
+    } else if (e instanceof WebClientResponseException.NotImplemented) {
+      return Mono.error(new WebClientResponseWithMessageException((WebClientResponseException) e));
```

Comment:
Instead of having a separate block we can combine this case with the existing block from line 303 - 306 as same exception is thrown.

Reply:
we could! but the code becomes bit verbose:
```
else if ((e instanceof WebClientResponseException.NotImplemented) || (e instanceof WebClientResponseException
        && ((WebClientResponseException) e).getStatusCode().is4xxClientError())) {
```
I prefer current way, do you feel strongly?

Reply:
seems non-blocking for now, resolving

Reply:
Sounds good.